lib/rubocop/cop/style/redundant_ternary.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+# TODO: when finished, run `rake generate_cops_documentation` to update the docs
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for redundant uses of true/false in ternary conditions
+      #
+      # @example
+      #   # bad
+      #   x == y ? true : false
+      #
+      #   # good
+      #   x == y
+      #
+      #   # bad
+      #   x == y ? false : true
+      #
+      #   # good
+      #   x != y
+      class RedundantTernary < Cop
+        include ConfigurableEnforcedStyle
+
+        OPS = %w[== === != < > <= >= <=>].freeze
+        MSG = 'Returning true/false from a conditional is unnecessary'.freeze
+
+        def on_if(node)
+          return unless node.ternary? && offense?(node)
+
+          add_offense(node, node.source_range, MSG)
```

Comment:
`MSG` is passed automagically to `#add_offense` if you omit it. There are also keywords available for the second argument, giving:

```ruby
add_offense(node, :expression)
```